README.md

```diff
@@ -19,12 +19,15 @@ checked in, and you must make sure it is up to date when you push commits to
 this repository.
 
 Code generation has the following dependencies:
-  - The protobuf compiler itself.  On ubuntu, this package is `protobuf-compiler`.
+  - The protobuf compiler itself. On ubuntu, this package is `protobuf-compiler` and can be installed through `sudo apt install protobuf-compiler`.
+  - The protobuf documentation generation plugin:
+    - `go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc`
   - The golang plugins:
     - `go get -u github.com/golang/protobuf/protoc-gen-go@v1.3.0`
     - `go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v1.8.2`
-  - The documentation generation plugin:
-    - `go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc`
+  - The python plugins:
+    - `pip install grpcio-tools`
```

Comment:
Is there a way to install specific versions, so that the output is reproducible?